File: legacy/src/main/java/com/redbox/domain/article/dto/CreateArticleRequest.java
package com.redbox.domain.article.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
public class CreateArticleRequest {

    @NotBlank(message = "제목을 입력해주세요.")
    private String subject;

    @NotBlank(message = "기사 URL을 입력해주세요.")
    private String url;

    @NotBlank(message = "출처를 입력해주세요.")
    private String source;
}

File: migration/src/main/kotlin/com/redbox/domain/community/article/dto/CreateArticleRequest.kt
package com.redbox.domain.community.article.dto

import jakarta.validation.constraints.NotBlank

data class CreateArticleRequest(
    @NotBlank(message = "제목을 입력해주세요.")
    val subject: String? = null,

    @NotBlank(message = "기사 URL을 입력해주세요.")
    val url: String? = null,

    @NotBlank(message = "출처를 입력해주세요.")
    val source: String? = null
) {
}

